frameworks/Java/servicetalk/src/main/java/servicetalk/Server.java

```diff
@@ -0,0 +1,100 @@
+import io.servicetalk.transport.api.ConnectionContext;
+import io.servicetalk.transport.api.IoExecutor;
+import io.servicetalk.transport.netty.internal.*;
+
+import static io.servicetalk.concurrent.api.Single.succeeded;
+
+
+
+public final class Server {
+
+
+    public static void main(String[] args) throws Exception {
+
+        /**
+         * Disable  AsyncContext
+         */
+        AsyncContext.disable();
+
+        /**
+        *   Factory to implement io pooling
+        */
+        IoExecutor ioExecutor = NettyIoExecutors.createIoExecutor(
+          Runtime.getRuntime().availableProcessors(),
+          new IoThreadFactory("io-pool")
+        );
+
+        /**
+         * Factory to disable headers validation
+         */
+
+        DefaultHttpHeadersFactory headersFactory = new DefaultHttpHeadersFactory(false,false);
+
+        try{
+            HttpSerializationProvider serializer = HttpSerializationProviders.jsonSerializer(new JacksonSerializationProvider());
+//            Create a custom server builder with performance enhancements
+            HttpServers
+              .forPort(8080)
+              .executionStrategy(HttpExecutionStrategies.noOffloadsStrategy())
+              .ioExecutor(ioExecutor)
+              .disableDrainingRequestPayloadBody()
+              .protocols(HttpProtocolConfigs.h1().headersFactory(headersFactory).build())
+              .appendConnectionAcceptorFilter(delegate -> new ConnectionAcceptor() {
+                  @Override
+                  public Completable accept(ConnectionContext context) {
+                      ((NettyConnectionContext)context).updateFlushStrategy((current, isOrig) -> FlushStrategies.flushOnEnd());
+                      return delegate.accept(context);
+                  }
+              })
+              .listenAndAwait((ctx, request, responseFactory) -> {
+                  ((NettyConnectionContext)ctx).updateFlushStrategy(((current, isCurrentOriginal) -> FlushStrategies.flushOnEach()));
+                  if(request.path().equals("/json")) {
+                      Map<String, String> obj = new HashMap<String, String>();
+                      obj.put("message", "Hello, World!");
+                      return succeeded(responseFactory.ok()
+                        .payloadBody(obj, serializer.serializerFor(Map.class))
+                        .addHeader("Date", GetCurrentTime())
+                        .addHeader("Server", "TFB"));
+                  }
+
+                  if(request.path().equals("/plaintext")) {
+                      return succeeded(responseFactory.ok()
+                        .payloadBody("Hello, World!", HttpSerializationProviders.textSerializer())
+                        .addHeader("Date", GetCurrentTime())
+                        .addHeader("Server", "TFB"));
+                    };
+                    return null;
+                })
+                .awaitShutdown();
+        } catch (Exception e) {
+//            logging is not allowed
```

Comment:
You might as well not have the `try` and `catch` then.  Your main method declared `throws Exception`, so no good comes out of catching this.  The only thing it can do is hide errors from us.